src/software/ai_vs_ai_ci.py

```diff
@@ -0,0 +1,90 @@
+from software.thunderscope.proto_unix_io import ProtoUnixIO
+from software.thunderscope.binary_context_managers import *
+from software.thunderscope.replay.proto_logger import ProtoLogger
+from proto.message_translation import tbots_protobuf
+import software.python_bindings as cpp_bindings
+
+import numpy
+
+
+def start_ai_vs_ai(simulator_runtime_dir, blue_fs_dir, yellow_fs_dir):
```

Comment:
we can't use thunderscope_main for this because pyqt doesn't play nicely with display-less devices

Reply:
this file is removed now